llvm/lib/Transforms/Coroutines/CoroSplit.cpp

```diff
@@ -79,6 +79,73 @@ using namespace llvm;
 
 namespace {
 
+// Created on demand if the coro-early pass has work to do.
+class Lowerer : public coro::LowererBase {
```

Comment:
Why do we need to inherit `coro::LowererBase`? Can't we make it directly in a function?

Reply:
Yes, it is possible. Switched to a simple function.